Apps/W1/EDocument/test/src/Receive/EDocReceiveTest.Codeunit.al

```diff
@@ -185,10 +185,8 @@ codeunit 139628 "E-Doc. Receive Test"
         EDocumentPage.OpenView();
         EDocumentPage.Last();
 
-        Assert.AreEqual(Format(Enum::"E-Document Service Status"::"Imported Document Created"), EDocumentPage.EdocoumentServiceStatus.Status.Value(), 'Wrong service status for processed document');
```

Comment:
why this change?

Reply:
just was checking something on running the tests, will restore this code